examples/with-fauna/pages/index.js

```diff
@@ -4,7 +4,6 @@ import cn from 'classnames'
 import formatDate from 'date-fns/format'
 import useSWR, { mutate, SWRConfig } from 'swr'
```

Comment:
The reason we should move is because you won't need SWR here, so you can abstract things to:

```
// lib/fauna.ts

import 'server-only'

// code code
```

That will ensure the code only ever runs on the server. `npm i server-only`. 

https://nextjs.org/docs/app/building-your-application/data-fetching/patterns#using-react-cache-server-only-and-the-preload-pattern